dev-tools/packer/version.yml

```diff
@@ -1 +1,2 @@
-version: "5.0.0"
+
```

Comment:
No, it was generated like that by `dev-tools/set_version`. I fixed it now in the script.